### DISCUSSION TITLE
Nginx-UI中的【管理stream】报错

### DISCUSSION THREAD

**Discussion by 971230** [0xJacky/nginx-ui#769] [Q&A] [answered]
我是使用的apt-get install nginx安装的，在之后直接使用官网的命令快捷安装了Nginx-UI，但是里面的【管理stream】页报错，用不了，报错提示是 【open /etc/nginx/streams-available: no such file or directory】，是我们的Nginx-UI问题么？默认安装的总不至于出错吧。如果说是缺少东西，这个Nginx是少啥了？大致怎么添加啊，诶。
![image](https://github.com/user-attachments/assets/0a833400-bc70-4ef2-8c22-d1382de3a587)

**Comment by 0xJacky** [comment]
你好，这个是正常的，如果你没有用到 Stream 转发，可以不管这个错误。

**Reply by 971230** [reply]
我想多余问一下哈，我要使用Stream功能的话，nginx这样的安装方式还能拥有么？好像nginx的文件分布在系统各个地方，和自己编译安装的不一样了。。。。。。

**Reply by 0xJacky** [reply, marked as answer]
这个模块一般都有的，创建文件夹就可以使用，然后在 nginx.conf 里 include 一下

创建两个文件夹（例如，具体路径看你的配置）
```
/etc/nginx/streams-available
/etc/nginx/streams-enabled
```

```
stream {
	# ...
	include /etc/nginx/streams-enabled/*;
}
```

**Reply by 971230** [reply]
行，谢谢🙏大佬的提示👍👍👍
